src/pages/docs/git.md

```diff
@@ -0,0 +1,285 @@
+
+After you create the Pull Request, someone will review it. This can take up to 24h for important projects, and up to a week for everything not urgent.
+
+The reviewer can either approve your PR, make comments or request changes. We try to review PRs in a constructive and educational manner, so don't be upset if it takes a few tries before it is approved. You learn the most by people commenting on your PR and suggesting new and better approaches to your problems, not by coding a solution.
+
+## Checks: CI / CD
+
+Once you open your PR, you may notice a few automatic checks happening, like the following image.
+
+![CI checks in progress](/images/git/ci.png "CI checks in progress")
+
+ This checks are part of what's called *Continuous Integration / Continuous Delivery*, or CI/CD, for short. In order for your PR to be merged, these checks must pass. Normally there are 2/3 checks performed:
+
+- Code Quality: this checks if your code follows your style guidelines. If this task fails, you can fix it by running `npm run format` on NextJS projects and `mix format` on Elixir projects. `npm run lint` and `mix lint` might also be needed
+
+- Build and Test: this only applies to backend projects, and it checks to make sure all the tests in the codebase run successfully (and that the project compiles, of course). You can simulate this task by running `mix test` on your local machine. If any tests fail, you will need to fix them before merging
+
+- Deploy Previews: deploys your branch on a temporary URL for testing purposes. If all other tasks pass, this one should not be a problem and, if it is, it probably is something DevOps related, so contact the person in charge of that.
+
+
+## Rebases, Merges and Conflicts
+
+If may happen that, in the time you opened your PR and someone approved it, another PR was merged and it causes conflicts with your code. This would be indicated in Github like the following image.
+
+To fix this, you can do two things: rebase your branch so it is up to date with `main` - we recommend doing this 90% of the time - or merging your branch with `main` - which we only recommend if the rebase process would be very difficult.
+
+### Rebase
+
+To rebase your branch start by updating your `main` branch by running:
+
+```
+git switch main
+git pull
+```
+
+Now switch back to your branch and start the rebase:
+
+```
+git switch <YOUR-BRANCH>
+git rebase main
+```
+
+Now go to your favourite text editor, and fix the conflicts yourself. Once you are done, add all the files and continue with the rebase.
+
+```
+git add <FILES CHANGED>
+git rebase --continue
+```
+
+You might have to repeat this a few times before the rebase succeeds. When it does, it is time to push your branch to the remote. However, a rebase is a rewriting of history, meaning you will have to force push by running
+
+```
+git push -f
+```
+
+### Merge
+
+To merge your branch start by updating your `main` branch by running:
+
+```
+git switch main
```

Comment:
I'm not sure, but I don't think you need to switch to main before pulling

Reply:
You need to do it otherwise you are pulling the current branch